<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UriDialog</class>
 <widget class="QDialog" name="UriDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location of remote datasource</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Scheme</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxScheme">
     <item>
      <property name="text">
       <string>file:// (local filesystem)</string>
      </property>
      <property name="icon">
       <iconset resource="dlsgui.qrc">
        <normaloff>:/images/drive-harddisk.svg</normaloff>:/images/drive-harddisk.svg</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dls:// (network protocol)</string>
      </property>
      <property name="icon">
       <iconset resource="dlsgui.qrc">
        <normaloff>:/images/Network-idle.svg</normaloff>:/images/Network-idle.svg</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditHost">
     <property name="text">
      <string notr="true">localhost</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxPort">
     <property name="specialValueText">
      <string>default (53584)</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditPath"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>URI</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditUri"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="dlsgui.qrc"/>
 </resources>
 <connections/>
</ui>
